python_modules/dagster/setup.py

```diff
@@ -81,10 +98,9 @@ def get_version() -> str:
         # pin around issues in specific versions of alembic that broke our migrations
         "alembic>=1.2.1,!=1.6.3,!=1.7.0,<1.11.0",
         "croniter>=0.3.34",
-        f"grpcio>={GRPC_VERSION_FLOOR},<{GRPC_VERSION_CAP}; python_version<'3.11'",
-        f"grpcio>={GRPC_VERSION_FLOOR}; python_version>='3.11'",
-        f"grpcio-health-checking>={GRPC_VERSION_FLOOR},<{GRPC_VERSION_CAP}; python_version<'3.11'",
-        f"grpcio-health-checking>={GRPC_VERSION_FLOOR}; python_version>='3.11'",
+    ]
+    + get_grpcio_requires()
```

Comment:
@gibsondan I don't think this will work.

This will work look at the env var when we build the wheel, eg [here](https://github.com/dagster-io/internal/blob/e846c750f30c86bc9851f5bdfc91846d53d921c2/python_modules/automation-internal/automation_internal/release/cli.py#L271).

But not during `pip install` which will just use the wheel and not run setup.py.

We can only do this if we ask users to build from the source (eg run `python setup.py install`)